samples/openapi3/client/petstore/python-experimental/tests/test_deserialization.py

```diff
@@ -176,3 +176,27 @@ def test_deserialize_fruit_null_value(self):
 
         inst = petstore_api.FruitReq(None)
         self.assertIsNone(inst)
+
+    def test_deserialize_all_of_with_additional_properties(self):
+        """
+        deserialize data.
+        """
+
+        # Dog is allOf with two child schemas.
+        # The OAS document for Dog does not specify the 'additionalProperties' keyword. 
+        # The additionalProperties keyword is used to control the handling of extra stuff,
+        # that is, properties whose names are not listed in the properties keyword.
+        # By default any additional properties are allowed.
+        data = {
+            'className': 'Dog',
+            'color': 'brown',
+            'breed': 'golden retriever',
+            # Below are additional, undeclared properties
+            'group': 'Terrier Group',
```

Comment:
@spacether , currently this unit test raises the following exception:

`petstore_api.exceptions.ApiValueError: Invalid input arguments input when making an instance of class Dog. Not all inputs were used. The unused input data is {'group': 'Terrier Group', 'size': 'medium'}`

Reply:
Does the java code set additionalProperties to true? I think that we should fix that on the java side, then the python will work.

Reply:
The oas document spec version should determine this additionalProperties behavior

Reply:
> Does the java code set additionalProperties to true? I think that we should fix that on the java side, then the python will work.

ok, let me check the Java side.

Reply:
There is a good discussion at https://stackoverflow.com/questions/22689900/json-schema-allof-with-additionalproperties

Reply:
Yes, this is a problem in Java codegen: https://github.com/OpenAPITools/openapi-generator/pull/6290/files#diff-57d7532cf464a8d7c24aab4b22ceb993R1102